protocol-units/bridge/integration-tests/tests/client_eth_movement.rs

```diff
@@ -456,3 +459,103 @@ async fn test_eth_client_should_successfully_complete_transfer() {
 
 	//TODO: Here call complete with the id captured from the event
 }
+
+#[tokio::test]
+async fn test_eth_client_lock_then_complete_transfer() -> Result<(), anyhow::Error> {
```

Comment:
This test is failing for me against a local Suzuka node. What test do you recommend for confirming that the new Eth monitoring works @musitdev ? The E2E Eth -> Movement test against the framework?

Reply:
The e2e framework test is passing for me for framework, I'm keen to get this into production so going to resolve this hope you don't mind